integrations/purchase/bridge-api-categories.js

```diff
@@ -0,0 +1,203 @@
+import {
+  ACTIVITY_TYPE_PURCHASE,
+  PURCHASE_CATEGORY_TRANSPORTATION_FUEL,
+  PURCHASE_CATEGORY_TRANSPORTATION_AUTOMOTIVE_MAINTENANCE_AND_REPAIR,
+  PURCHASE_CATEGORY_STORE_BOOKS,
+  PURCHASE_CATEGORY_STORE_CLOTHING,
+  PURCHASE_CATEGORY_STORE_FOOD,
+  PURCHASE_CATEGORY_HEALTHCARE_DOCTOR,
+  PURCHASE_CATEGORY_MEDICINES_AND_HEALTH_PRODUCTS,
+  PURCHASE_CATEGORY_STORE_GARDEN_AND_PET,
+  PURCHASE_CATEGORY_STORE_FURNISHING,
+  PURCHASE_CATEGORY_STORE_PERSONAL_CARE,
+  PURCHASE_CATEGORY_STORE_HARDWARE,
+  PURCHASE_CATEGORY_FINANCIAL_SERVICES,
+  PURCHASE_CATEGORY_SOCIAL_PROTECTION,
+  PURCHASE_CATEGORY_INSURANCE,
+  PURCHASE_CATEGORY_RECREATIONAL_SERVICES,
+  PURCHASE_CATEGORY_INFORMATION_COMMUNICATION,
+  PURCHASE_CATEGORY_FOOD_SERVING_SERVICES,
+  PURCHASE_CATEGORY_TRANSPORT_AIR,
+  PURCHASE_CATEGORY_TRANSPORT_RAIL,
+  PURCHASE_CATEGORY_TRANSPORT_ROAD,
+  PURCHASE_CATEGORY_RECREATIONAL_DURABLES,
+  PURCHASE_CATEGORY_NON_DURABLE_HOUSEHOLD_GOODS,
+  PURCHASE_CATEGORY_MISC_DWELLING,
+  PURCHASE_CATEGORY_READY_FOOD,
+  PURCHASE_CATEGORY_RENTAL,
+  PURCHASE_CATEGORY_COMBINED_PASSENGER_TRANSPORT,
+  PURCHASE_CATEGORY_MISC_SERVICES_PERSONAL_TRANSPORT,
+  PURCHASE_CATEGORY_HOUSING,
+  PURCHASE_CATEGORY_ELECTRICITY,
+  PURCHASE_CATEGORY_GAS,
+  PURCHASE_CATEGORY_HOUSEHOLD_MAINTENANCE,
+  PURCHASE_CATEGORY_ENTERTAINMENT_HOTEL,
+  PURCHASE_CATEGORY_ENTERTAINMENT_CIGAR_STORES,
+  PURCHASE_CATEGORY_PACKAGE_HOLIDAYS,
+} from '../../definitions';
+
+const purchaseActivity = purchaseType =>
+  purchaseType
+    ? {
+        purchaseType,
+        activityType: ACTIVITY_TYPE_PURCHASE,
+      }
+    : null;
+
+export const BRIDGE_API_CATEGORIES = {
+  303: purchaseActivity(), // - Withdrawals, checks & transfer
```

Comment:
Same comment as for the nag categories: `303: null` would be more clear what is going on.